testcases/setup-teardown-5.test.js

```diff
@@ -0,0 +1,19 @@
+export default {
+  title: 'Dummy - Setup / Teardown - 5',
+  description: 'Testing to see if the Teardown is called when the test fails',
+  setup() {
+    this.$log('I am a synchronous Setup method. I should run before the steps')
+  },
+  teardown() {
+    this.$log('I am a synchronous Teardown method that fails and is ignored')
+    throw new Error('This is a thrown error')
+  },
+  steps: [
+    {
+      description: 'The same synchronous test with assert that fails',
+      test: x => x,
+      params: 1,
+      assert: 1,
```

Comment:
Here the test always pass as the params and assert are same. I think the test should be failed and then check the teardown

Reply:
Correct, the description is incorrect and needs to be aligned.